packages/settingeditor/style/base.css

```diff
@@ -444,16 +461,32 @@ ul.jp-PluginList li.jp-mod-selected span.jp-PluginList-icon.jp-FileIcon {
   margin: 8px 12px 0;
 }
 
+.jp-PluginList mark {
+  background-color: transparent;
+  font-weight: bold;
+  color: var(--jp-ui-font-color1);
+}
+
 .jp-PluginList-entry {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   border: 1px solid transparent;
   background: transparent;
   overflow: hidden;
   padding: 4px 0 4px 4px;
   text-overflow: ellipsis;
```

Comment:
In addition to the suggestion in https://github.com/jupyterlab/jupyterlab/pull/12082/files#r830543410, this `text-overflow: ellipsis;` needs to be moved to the span element which is inside of `.jp-pluginList-entry-label` or otherwise it has no effect.

Let's add:

```css
.jp-pluginList-entry-label span {
  text-overflow: ellipsis;
  overflow-x: hidden;
  white-space: nowrap
}
```

and remove it from here.

```suggestion
```

| Before (ellipsis has no effect, `jp-SelectedIndicator` is squashed) | After |
| --- | --- |
| ![Screenshot from 2022-03-20 00-54-46](https://user-images.githubusercontent.com/5832902/159143499-df8a6022-9344-4043-90ed-34a3d7dc57d4.png) | ![Screenshot from 2022-03-20 00-53-54](https://user-images.githubusercontent.com/5832902/159143500-2dfcea07-0803-4199-8ab2-241376f90cb7.png) |